include/ck_tile/ops/gemm/kernel/gemm_kernel.hpp

```diff
@@ -19,13 +19,13 @@ template <typename TilePartitioner_,
           typename LayoutC_>
 struct GemmKernel
 {
-    using TilePartitioner                    = remove_cvref_t<TilePartitioner_>;
-    using GemmPipeline                       = remove_cvref_t<GemmPipeline_>;
-    using EpiloguePipeline                   = remove_cvref_t<EpiloguePipeline_>;
-    using LayoutA                            = remove_cvref_t<LayoutA_>;
-    using LayoutB                            = remove_cvref_t<LayoutB_>;
-    using LayoutC                            = remove_cvref_t<LayoutC_>;
-    static constexpr index_t KernelBlockSize = GemmPipeline::KernelBlockSize;
+    using TilePartitioner               = remove_cvref_t<TilePartitioner_>;
+    using GemmPipeline                  = remove_cvref_t<GemmPipeline_>;
+    using EpiloguePipeline              = remove_cvref_t<EpiloguePipeline_>;
+    using LayoutA                       = remove_cvref_t<LayoutA_>;
+    using LayoutB                       = remove_cvref_t<LayoutB_>;
+    using LayoutC                       = remove_cvref_t<LayoutC_>;
+    static constexpr index_t kBlockSize = GemmPipeline::kBlockSize;
```

Comment:
LGTM. The only thing is it better keep the name of KernelBlockSize in the gemm_kernel.hpp other place could keep as kBlockSize. Thank you for the fix!